bittensor/__init__.py

```diff
@@ -207,7 +209,9 @@ def turn_console_off():
 from bittensor._dendrite.text_prompting.dendrite_pool import TextPromptingDendritePool as text_prompting_pool
 
 # ---- Base Miners -----
-from bittensor._synapse.text_prompting.miner import BasePromptingMiner
+from bittensor._neuron.base_miner_neuron import BaseMinerNeuron as base_miner_neuron
```

Comment:
small nit: BasePromptingMiner and base_miner_neuron are two separate conventions. We should stick with one, ie BaseMinerNeuron.

Reply:
Fixed with commit: https://github.com/opentensor/bittensor/commit/07fad43a72bcdcd3c941ef84ca39c72db2e1c9a3